engines/director/lingo/lingo-the.cpp

```diff
@@ -519,7 +519,7 @@ Datum Lingo::getTheEntity(int entity, Datum &id, int field) {
 		d = (int)(_vm->getMacTicks() - movie->_lastEventTime);
 		break;
 	case kTheLastFrame:
-		d = (int)score->_frames.size() - 1;
+		d = (int)score->getTotalFrames() - 1;
```

Comment:
Need to check if it is truly `- 1`. Because the frames originally start numbered from 1.